### ISSUE TITLE
How to use are.na workshop

### ISSUE THREAD

**Issue by callil** [new-computers/workshops#1]
Host a meet up to share methods for are.na organization. Figure out how to map out some fundamental research tactics and structures to teach new or even experienced users. Outcome of meet up should be published documentation and guides.

**Comment by urcades** [comment]
Brainstorming for ourselves "common" use cases seems to be an OK first step.

For my own purposes, I started using [Kingfisher](https://kingfisher.link/maps/98IE9PSYT/edit/NF23ZEOE2) to loosely capture my own behaviors with regards to reading, highlighting text/passages/quotes, and adding these highlights to are.na — these overall flows inform how I go about populating my 'Reading Networks' chennels, which might be useful (as an example) for new folks to see.

**Comment by jmandel1027** [comment]
I think something to consider is how we're utilizing an archive as a companion to represent larger bodies of thought/work. I agree w/ Ed for thinking about common use cases to work off as starting points, ie are we scraping for text modeling, or using it as an aggregator for a tactile mood board with drag-able and nest-able content posting/pulling to/from Arena. Deff into this! Sounds like a lot of fun and lead to other interesting projects. n_n

Another Idea I had is how we can use things like firebase as an ambient task cache. I've been playing with passing live bio data for some biofeedback synths with arduino and p5 lately with a lil python script writing the signals from arduino to firebase w/ p5 listening. This could be fun for peeps of all levels.

**Comment by jacoblindgren** [comment]
This all sounds great, just getting around to looking through the Kingfisher doc. I'd like to think are.na is as easy or easier to operate than some of the popular social media websites/their apps, but is there any room for some a sort of ground-up, basic walkthrough intended for those who may not be accustomed to the web as much as some of the other texts suggest? Also, in terms of a common case, is there something that someone could do that doesn't rely on having a pre-existing idea or body of work that they want to explore?

**Comment by as-dr** [comment]
I agree with @edouerd that the best first step is to clarify and outline a collection of common techniques and activities and _then_ offer guides for how those can be undertaken & completed within the scope of Are.na's features / context.

Taking a leaf from his Kingfisher map, we could organize them into distinct categories e.g.:

1. Researching
2. Collecting
3. Synthesizing
4. Organizing
5. Learning
6. Etc etc.

**Comment by jacoblindgren** [comment]
I like the sound of and agree with all of those categories. Not necessarily immediately relevant to our scope but is it clear to people looking at a channel which of those categories it is being used for? Not that it would need to be, I just find it interesting how they're all likely subjective once anyone other than the creator of the channel is viewing it.

**Comment by as-dr** [comment]
To clarify, I wasn't necessarily advocating for these specific categories, but for categorization more generally. I love how Are.na articulates itself as a "Platform for Collaborative Research" but that's appealing to *me* (and likely to everyone else here) but may not be to others, who may be discouraged by the phrasing.

To those people (whom I suspect are a much larger demographic) my thought is that categorization provides:

1. Multiple clear, relatable use cases
2. The steps to achieve those use cases
3. An alternative, user-created onboarding guide.
4. An approach to minimize time-to-value.

You raise an interesting point. Perhaps emphasizing the actions implicit in a particular use case over their output (in this case, a channel) is the way to establish an objective "baseline" to operate from.
